### PR TITLE
add minimuxer_set_location for set ios location

### PR DESCRIPTION
**Pull request by sentry5678**

### CODE REVIEW

File: src/location.rs [hunk 1]
```diff
@@ -0,0 +1,173 @@
+
+use log::{error, trace};
+use crate::{errors::Errors, fetch_first_device, test_device_connection};
+use rusty_libimobiledevice::{ service::ServiceClient};
+
+#[no_mangle]
+/// is device connect
+/// # Safety
+/// Don't be stupid
+pub unsafe extern "C" fn minimuxer_isconnect() -> libc::c_int {
```
Comment (amsam0) [inline src/location.rs:10]: Please see minimuxer_ready: https://github.com/SideStore/minimuxer/pull/4/files#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759R79

**Review by amsam0 — CHANGES_REQUESTED** [review]
Thanks for the contribution! However, I’m a bit confused why we would need this function. minimuxer is only supposed to be used in SideStore. If you want to use it outside of SideStore, please fork minimuxer instead of making a PR.

If this is intended to be used in SideStore, what functionality would it serve? I don’t see the use in manually setting location.

**Comment by amsam0** [comment]
Hey @hw-1, I discussed with some other team members and we are planning to add this as a feature eventually, but I think this PR will need some code cleanup first. It will also need an implementation in SideStore, but I may be able to do that.

**Review by silxl — COMMENTED** [review]
Access

**Pull request closed without merging**